BoilerPlates/javascript/README.md

```diff
@@ -0,0 +1,90 @@
+# Simple Chart with Webpack and NPM
+
+This is the simplest project using SciChart.js as npm module and Webpack.
+
+## Trial licensing
```

Comment:
Replace this with the more up to date licensing section from the scichart-react readme.  Same for the other boilerplates